src/Cake.NuGet/NuGetPackageInstaller.cs

```diff
@@ -175,6 +160,38 @@ public IReadOnlyCollection<IFile> Install(PackageReference package, PackageType
             return result;
         }
 
+        private static DirectoryPath GetPackagePath(DirectoryPath root, PackageReference package)
+        {
+            if (package.Parameters.ContainsKey("version"))
+            {
+                var version = package.Parameters["version"].First();
+                return root.Combine($"{package.Package}.{version}");
```

Comment:
Well, not that easy. The `NuGetVersion` representation of `0.6.0` => `0.6.0.0`, but that's not the install folder. Same with `2.4.5` => `2.4.5.0`. Might be different with pre-release packages as well. I'll take a look.